dev-resources/bootstrapping/cli/bootstrap_with_comments.cfg

```diff
@@ -0,0 +1,5 @@
+# commented out line
+puppetlabs.trapperkeeper.examples.bootstrapping.test-services/hello-world-service # comment
+; another commented out line
+ ;puppetlabs.trapperkeeper.examples.bootstrapping.test-services/foo-test-service
```

Comment:
I just copied this from a test, but it seemed intentional, to prove that it could handle lines with spaces and comments I guess?